tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "module": "commonjs",
+    "target": "es6",
+    "jsx": "preserve", 
+  },
+  "allowJs": false,
```

Comment:
This value up till the 'include' (excluding the include prop) are all child properties of `compilerOptions`.